src/EventStore.Core/ClusterVNodeOptions.cs

```diff
@@ -270,9 +270,14 @@ public record CertificateFileOptions {
 			[Description("The password to the certificate if a PKCS #12 (.p12/.pfx) certificate file is provided."),
 			 Sensitive]
 			public string? CertificatePassword { get; init; }
+			
+			[Description("The password to the certificate private key if an encrypted PKCS #8 private key file is provided."),
+			 Sensitive]
+			public string? PrivateKeyPassword { get; init; }
```

Comment:
```suggestion
			public string? CertificatePrivateKeyPassword { get; init; }
```

suggestion: for consistency with other certificate options, we can rename it to `CertificatePrivateKeyPassword`